cmd/api/src/database/audit.go

```diff
@@ -78,5 +81,27 @@ func (s *BloodhoundDB) ListAuditLogs(before, after time.Time, offset, limit int,
 		result = cursor.Find(&auditLogs).Count(&count)
 	}
 
+	// Populate the actor name and email from the users table. Uses a map as a cache to avoid looking up the same user multiple times.
+	// NOTE: This is intended to be a temporary solution for including this data directly in the returned AuditLog. This is due to a lack
+	// of flexibility with gorm's associations/JOINs.
+	// TODO: When gorm is removed, select these values with a JOIN on the users table.
+	for i, log := range auditLogs {
+		var actor model.User
```

Comment:
There's no real need to declare actor here since we aren't worried about shadowing and we have to do `actor, ok := ...` anyway. It doesn't hurt anything but is a little redundant

Reply:
Oh good catch. That was leftover from a small refactor. I'll clean it up.